run_node.sh

```diff
@@ -31,19 +38,38 @@ get_version () {
     # The version that is returned by the extrinsic looks like: 0.11.4-ae34eb4213
     # so we take the hash part and use it to identify docker images and commits in the repo.
 
-    if  [ -n "${MAINNET}" ]
+    if [ -n "${VERSION}" ]
```

Comment:
Any reason why we use sometimes `if [ ]` and `if [[ ]]` in this script? E.g. this line and line 49th